implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -34,14 +50,43 @@ private static void getDefaultFromClasspath(File propertyFile) {
         }
     }
 
+    private static PropertiesConfiguration getReferenceProperties(File propertyFile) {
```

Comment:
Reference properties file is a file from classpath (resource/jbb-*.properties) with all needed properties